traits/ctraits.c

```diff
@@ -4885,37 +4885,124 @@ static PyMethodDef trait_methods[] = {
 |  'CTrait' property definitions:
 +----------------------------------------------------------------------------*/
 
+PyDoc_STRVAR(
+    ctrait_handler_doc,
+    "The trait handler underlying this trait.\n"
+    "\n"
+    "The value of this property should be an instance of\n"
+    "``BaseTraitHandler``.\n"
+);
+
+PyDoc_STRVAR(
+    ctrait_post_setattr_doc,
+    "Callable called after a successful value assignment to this trait.\n"
+    "\n"
+    "The value of this property is either a callable or *None*.\n"
+    "If the value is a callable, this callable allows the trait to do\n"
+    "additional processing after a value has successfully been assigned.\n"
+    "The callable is called with arguments (object, name, value), and the\n"
+    "return value of the callable is ignored.\n"
+);
+
+PyDoc_STRVAR(
+    ctrait_is_property_doc,
+    "True if this trait is a property trait, else False.\n"
+    "\n"
+    "This property is read-only.\n"
+);
+
+PyDoc_STRVAR(
+    ctrait_modify_delegate_doc,
+    "Indicate whether modifications affect the delegate.\n"
+    "\n"
+    "For delegated traits, this is a boolean indicating whether\n"
+    "modifications to this trait also modify the delegated trait.\n"
+);
+
+PyDoc_STRVAR(
+    ctrait_setattr_original_value_doc,
+    "Whether setattr stores the original or the validated value.\n"
+    "\n"
+    "If true, setattr will store the original, unvalidated, value set on\n"
+    "the trait to the object's dictionary. If false, the value returned\n"
+    "from the validator will be stored.\n"
+);
+
+PyDoc_STRVAR(
+    ctrait_post_setattr_original_value_doc,
+    "Whether post_setattr receives the original or the validated value.\n"
+    "\n"
+    "If true, the post_setattr callable for this trait (if defined)\n"
+    "receives the original, unvalidated value assigned to the trait.\n"
+    "If false, the validated value is provided to post_setattr.\n"
+);
+
+PyDoc_STRVAR(
+    ctrait_is_mapped_doc,
+    "True if this is a mapped trait, else False.\n"
+);
+
+PyDoc_STRVAR(
+    ctrait_comparison_mode_doc,
+    "Integer constant indicating when notifiers are executed.\n"
+    "\n"
+    "Valid values for this constant are 0, 1 and 2. If 0, notifiers\n"
```

Comment:
There is an Enum for this in `traits.constants.ComparisonMode`.  It is probably better to refer to that for the meanings.  That way there is only one place to update should the property change.

Reply:
Agreed, and done.